src/neuroconv/utils/json_schema.py

```diff
@@ -102,6 +102,9 @@ def get_json_schema_from_method_signature(method: Callable, exclude: Optional[li
     exclude = exclude or []
     exclude += ["self", "cls"]
 
+    split_qualname = method.__qualname__.split(".")[-2:]
+    method_display = ".".join(split_qualname) if "<" not in split_qualname[0] else method.__name__
```

Comment:
What is the `<` case for?

Reply:
Sometimes `<locals>` (local namespaces domain) shows up for basic functions. Think it depends on how those functions are defined/imported, but it happens in the tests and I think would happen if you locally defined a data interface in the same script you are trying to run this with